src/components/kompas-paywall-body/kompas-paywall-body.tsx

```diff
@@ -185,7 +187,17 @@ export class KompasPaywallBody {
     </div>
   )
   private separatorLine = (): void => (
-    <div class="hidden border-b-2 border-blue-200 w-1/4 my-4 md:flex justify-center" />
+    <div class="flex flex-row w-full justify-center">
+      <div class="border-b-2 border-blue-200 w-1/4 my-4 flex justify-center" />
+      <p class="px-4 pt-1">atau</p>
+      <div class="border-b-2 border-blue-200 w-1/4 my-4 flex justify-center" />
+    </div>
+  )
+  public subscribeWithGoogleSection = (): void => (
+      <button class="border-2 bg-grey-100 border-grey-100 rounded-md px-6 shadow-sm flex flex-row py-3 mt-1 mb-4">
+        <p>Subscribe with</p>
+        <div class="pl-2" innerHTML={GoogleLogo}></div>
```

Comment:
jangan pake innerhtml mas, not recommended soalnya ini ditaro di web component user bebas akses, takut kena xss ntar